docs/pages/access-controls/guides/device-trust.mdx

```diff
@@ -193,9 +192,9 @@ $ tsh logout; tsh login --proxy=example.com --user=alice # fetch new certificate
 
 The device above is now a trusted device.
 
-## Optional: Enforce trusted devices
+## Optional: Cluster-wide device enforcement
 
-Device trust has the following modes of operation, represented by the
+Cluster-wide configuration enforces trusted device access at cluster level. It can be configured with the following three modes of operation, represented by the
 `device_trust.mode` authentication setting:
```

Comment:
Hard break around 80 chars, and minor grammar edits:

```suggestion
Cluster-wide configuration enforces trusted device access at the cluster level.
It can be configured with the following three modes of operation, represented
by the `device_trust.mode` authentication setting:
```